ops/production-deploy.tmpl.yaml

```diff
@@ -0,0 +1,36 @@
+nextAuthUrl: "https://phenovista.softserv.cloud/api/auth"
```

Comment:
is this temporary? (along with the other references to it below) they want the site to point at store.phenovista.com.

ref: [this ticket](https://github.com/scientist-softserv/phenovista-digital-storefront/issues/17)

Reply:
yes, I dont have access to that url so we need something to get the deploy all figured out